daemon/api.go

```diff
@@ -645,6 +616,8 @@ var snapstateUpdate = snapstate.Update
 var snapstateInstallPath = snapstate.InstallPath
 var snapstateTryPath = snapstate.TryPath
 var snapstateGet = snapstate.Get
+var snapstateUpdateMany = snapstate.UpdateMany
```

Comment:
Can we please put that under a common var ( ... ) block, and sort so UpdateMany is under Update?